python_src/src/lamp_py/performance_manager/l0_rt_vehicle_positions.py

```diff
@@ -158,7 +158,7 @@ def transform_vp_timestamps(
         vehicle_positions,
         index=trip_stop_columns,
         columns="is_moving",
-        aggfunc={"vehicle_timestamp": min},
+        aggfunc={"vehicle_timestamp": "min"},
```

Comment:
thats kind of a strange change. was it using pythons built in min function before and is now using some internal one?

Reply:
Just having `min` still works, but throws a warning.  All of the pandas documentation for the `pivot_table` function references all agg functions as just strings.